crates/payload/builder/src/optimism.rs

```diff
@@ -110,7 +110,7 @@ impl PayloadBuilderAttributes for OptimismPayloadBuilderAttributes {
         let blob_excess_gas_and_price = parent
             .next_block_excess_blob_gas()
             .or_else(|| {
-                if spec_id == SpecId::CANCUN {
+                if spec_id >= SpecId::CANCUN {
```

Comment:
there should be `is_enabled` or similar that does this

I'd like to use the function